README.md

```diff
@@ -2,32 +2,42 @@
 
 This repository contains the snap package sources for [Tailscale](https://github.com/tailscale/tailscale).
 
-## Build the snap
-
+## Local development
 
 You can build the snap locally with:
 
 ```
 snapcraft --use-lxd
```

Comment:
is this correct command? or this one `snapcraft pack`

Reply:
Yep this is correct. :)

Reply:
yep, note `charmcraft` does not equal to `charmcraft pack`